clifford/numba/_multivector.py

```diff
@@ -54,6 +54,9 @@ def _numba_type_(self):
     cache = layout_type._cache
     dt = self.value.dtype
 
+    if not self.value.flags.c_contiguous:
+        raise ValueError('Clifford currently only supports JITing functions of MultiVectors with C_CONTIGUOUS value arrays.')
```

Comment:
```suggestion
        raise ValueError("Only clifford.MultiVector objects with contiguous coefficients can be passed to JITted functions")
```
since for 1d arrays, all contiguous arrays are c-contiguous